app/assets/stylesheets/application.scss

```diff
@@ -4,8 +4,12 @@
 @import 'colours';
 @import 'measurements';
 @import 'typography';
+@import 'shims';
 
 // pages specific view imports
 @import "views/case-studies";
 @import "views/unpublishing";
 
+// HELPERS
+// Styles for comment page elements
+@import "helpers/_available_languages";
```

Comment:
You don't need the leading underscore on the import line as Sass infers it as it is trying to load a partial file. The leading underscore just indicates to Sass that the file shouldn't be compiled by itself. It can just read `@import "helpers/available_languages";`.  To match the file naming convention of the other files in this project can the helper be named `helpers/_available-languages.scss` (with a dash for word separation).